test.sh

```diff
@@ -88,8 +89,25 @@ if [[ $OS != "fedora" ]]; then
 fi
 if [[ $PYTHON_VERSION -gt 2 ]]; then $RUN $PIP install -r requirements-py3.txt; fi
 
+case ${ACTION} in
+"test")
+  TEST_CMD="py.test --cov osbs --cov-report html -vv tests"
+  ;;
+"pylint")
+  # This can run only at fedora because pylint is not packaged in centos
```

Comment:
There's a python-pylint package available in EPEL7, but since that's also python 2.7 (with an older pylint version), I don't see any value in including it here.